### DISCUSSION TITLE
有没有什么方法只替换剧情、对话文本？

### DISCUSSION THREAD

**Discussion by impalep0** [GpointChen/FFXIVChnTextPatch-GP#149] [Q&A]
原因出于有时候汉化会和xivlauncher的补丁冲突，在板子上买卖东西时也很难以英文直接翻译过去

**Comment by GpointChen** [comment]
留下資料夾和Achievement.csv即可（如下所選）。資料夾內為各類任務文本，Achievement.csv為供檢測用。

![image](https://user-images.githubusercontent.com/34565875/163918663-9a819e25-4d93-4d00-b3fd-02ddf5cb3a1c.png)

**Reply by impalep0** [reply]
感谢回复，请问是直接从FFXIVChnTextPatch>resource>rawexd 那里直接删掉图中未选的文件就好还是其他地方有英文的backup需要将汉化的文件替换回英文？

**Reply by GpointChen** [reply]
直接刪除未選的文件即可

後面那半句我看不太懂是什麼意思
